cmd/contour/servecontext.go

```diff
@@ -82,10 +82,10 @@ type serveContext struct {
 	// DisableLeaderElection can only be set by command line flag.
 	DisableLeaderElection bool
 
-	// Should Contour register to watch the new service-apis types?
+	// Should Contour register to watch the new gateway-api types?
 	// By default this value is false, meaning Contour will not do anything with any of the new
 	// types.
-	// If the value is true, Contour will register for all the service-apis types
+	// If the value is true, Contour will register for all the gateway-api types
 	// (GatewayClass, Gateway, HTTPRoute, TCPRoute, and any more as they are added)
```

Comment:
Not directly related to this PR, but this comment is misleading - we should update it to indicate this field is now completely ignored.